Packs/DSPM/Integrations/DSPM/README.md

```diff
@@ -0,0 +1,946 @@
+            },
+            {
+                "Key": "AWS Secret Key",
+                "No": 3
+            },
+            {
+                "Key": "Credit Card Expiration Date",
+                "No": 4
+            },
+            {
+                "Key": "Certificate",
+                "No": 5
+            }
+        ]
+    }
+}
+```
+
+#### Human Readable Output
+
+>### Data Types
+> | No | Key  |
+> |----|------|
+>| 1  | ID Number - Aadhaar (India) |
+>| 2  | Artifactory API Key |
+>| 3  | AWS Secret Key |
+>| 4  | Credit Card Expiration Date |
+>| 5  | Certificate |
+
+### dspm-list-labels
+
+***
+Returns a list of label names based on the company.
+
+#### Base Command
+
+`dspm-list-labels`
+
+#### Input
+
+There are no input arguments for this command.
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| DSPM.Label.Key | String | Label key. | 
+| DSPM.Label.No | unknown | Label number. | 
+
+#### Command example
+```!dspm-list-labels```
+#### Context Example
+```json
+{
+    "DSPM": {
+        "Label": [
+            {
+                "Key": "PCI",
+                "No": 1
+            },
+            {
+                "Key": "PHI",
+                "No": 2
+            },
+            {
+                "Key": "PII",
+                "No": 3
+            },
+            {
+                "Key": "Confidential",
+                "No": 4
+            },
+            {
+                "Key": "Sensitive",
+                "No": 5
+            }
+        ]
+    }
+}
+```
+
+#### Human Readable Output
+
+>### Data Types
+> | No | Key  |
+> |----|------|
+>| 1  | PCI |
+>| 2  | PHI |
+>| 3  | PII |
+>| 4  | Confidential |
+>| 5  | Sensitive |
+
+### dspm-list-data-types-findings
+
+***
+Retrieves a list of data type findings for the company.
+
+#### Base Command
+
+`dspm-list-data-types-findings`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| region_in | Comma-separated list of regions. | Optional | 
+| region_equal | Exact region. | Optional | 
+| cloud_provider_in | A comma-separated list of cloud providers. Possible values are: AWS, AZURE, GCP, SNOWFLAKE, FILE_SHARE, O365. | Optional | 
+| cloud_provider_equal | Exact cloud provider. Possible values are: AWS, AZURE, GCP, SNOWFLAKE, FILE_SHARE, O365. | Optional | 
+| service_type_in | Comma-separated list of service types. | Optional | 
+| service_type_equal | Exact service type. | Optional | 
+| lifecycle_in | Comma-separated list of life cycles. | Optional | 
+| projectId_in | A comma-separated list of project IDs. | Optional | 
+| projectId_equal | Exact project ID. | Optional | 
+| lifecycle_equal | Exact life cycle. | Optional | 
+| sort | Sorting criteria in the format: property,(asc\|desc). Default sort order is ascending. Multiple sort criteria are supported. | Optional | 
+| limit | The maximum number of data types findings to retrieve. Default is 50. | Optional | 
+
```

Comment:
In the above argument descriptions (to be consistent), use the following:
- A comma-separated list
- Where we use 'Exact" add "The" at the start. For eg The exact service type.